src/providers/commandsProvider.ts

```diff
@@ -94,9 +94,6 @@ function execCommand(currentRedPath: string, command: string, fileUri?: vscode.U
         filePath = fileUri.fsPath;
     }
 
-    if (filePath.indexOf(' ') > 0) {
```

Comment:
It shouldn't, to be honest. In any case, this's the only way to pass paths containing spaces and for that purpose I know it works on Windows, Ubuntu and macOS.